spikeinterface/toolkit/postprocessing/template_similarity.py

```diff
@@ -0,0 +1,27 @@
+
+import sklearn.metrics.pairwise
+
+
+def compute_template_similarity(waveform_extractor, method='cosine_similarity'):
```

Comment:
@samuelgarcia nice! I think we should also have this handle the `best_channels` and the intersection